### ISSUE TITLE
Theme Showcase: Remove Setup tab for new themes and in the current theme card.

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#58706]
Existing premium themes had an extra tab with setup instructions:
<img width="1446" alt="Screen Shot 2021-12-01 at 11 48 37 AM" src="https://user-images.githubusercontent.com/2124984/144277105-e2232f33-6005-4b7f-b577-1a0daf55d341.png">

Free themes do not:
<img width="1447" alt="Screen Shot 2021-12-01 at 11 50 30 AM" src="https://user-images.githubusercontent.com/2124984/144277294-5d084142-24c1-4c6c-8747-a7a5edcda794.png">

We need to remove it on the individual theme sheet and the current theme card:

<img width="1140" alt="Screen Shot 2021-12-01 at 11 52 02 AM" src="https://user-images.githubusercontent.com/2124984/144277565-30be8b84-374d-4891-8394-6db4536d1866.png">

**Comment by nelsonec87** [comment]
@sixhours What about this one?

![image](https://user-images.githubusercontent.com/3801502/145085992-9f108b58-4a7a-4d94-ab25-d7256caf3220.png)

May I remove it as well?

It seems the "Setup" option is shown only for premium themes

```
	const support = {
		label: translate( 'Setup' ),
		icon: 'help',
		getUrl: getThemeSupportUrl,
		hideForTheme: ( state, themeId ) => ! isThemePremium( state, themeId ),
	};
```

If it's not gonna be shown for premium, I can even remove this whole check.

**Comment by sixhours** [comment]
Yep! That can be removed as well. :)

**Comment by sixhours** [comment]
Closed by #58917
